src/metabase/db/setup.clj

```diff
@@ -185,4 +188,20 @@
 
 (db/set-default-db-connection! {:datasource data-source*})
 
-(db/set-default-jdbc-options! {:read-columns mdb.jdbc-protocols/read-columns})
+(reset! t2.honeysql/global-options
+        {:quoted       true
+         :dialect      ::application-db
+         :quoted-snake false})
+
+(reset! t2.jdbc/global-options
+        {:read-columns mdb.jdbc-protocols/read-columns
+         :label-fn     u/lower-case-en})
+
+(methodical/defmethod t2.pipeline/build :around :default
```

Comment:
People always ask me, why would you ever need an `:around` method? This is a good use case. Whenever Toucan 2 builds a query, we bind some Honey SQL options.